packages/task/src/browser/quick-open-task.ts

```diff
@@ -321,7 +324,9 @@ export class TaskConfigureQuickOpenItem extends QuickOpenGroupItem {
     }
 
     getLabel(): string {
-        return `${this.task._source}: ${this.task.label}`;
+        return this.task._source
```

Comment:
I guess for a configured task the name of the root folder will be used for displaying.
Please see [here](https://github.com/theia-ide/theia/blob/master/packages/task/src/common/task-protocol.ts#L49)
Is it expected behavior?

Reply:
I mean if you add some configuration to `tasks.json` file:
```
{
      "label": "Run my test task",
      "type": "shell",
      "command": "./task-long-running"
    }
```
For `Configure Tasks` the task is displayed :
![configured_tasks_error](https://user-images.githubusercontent.com/5676062/62543731-1f6ead00-b867-11e9-8609-5b5b2b84bd67.png)

For `Run Task` the same task is displayed:
![run_tasks](https://user-images.githubusercontent.com/5676062/62544068-c5bab280-b867-11e9-885f-92eea2c84aca.png)